modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -2289,25 +2292,23 @@ else if (grps.isEmpty())
             Set<UUID> bltNodeIds =
                 new HashSet<>(F.viewReadOnly(srvNodes, F.node2id(), (node) -> CU.baselineNode(node, clusterState)));
 
-            startSnpProc.start(snpFut0.rqId, new SnapshotOperationRequest(
-                snpFut0.rqId,
-                cctx.localNodeId(),
-                name,
-                snpPath,
-                grps,
-                bltNodeIds,
-                incremental,
-                incIdx,
-                onlyPrimary,
-                dump,
-                compress,
-                encrypt
-            ));
+            var snpOpReq = new SnapshotOperationRequest(
```

Comment:
Do not use `var` until you have great reason for that. Style guide for using `var` is here [1]. In current situation is better to use explicit type `SnapshotOperationRequest`. It's name is simple and the variable is used twice. 

[1] https://openjdk.org/projects/amber/guides/lvti-style-guide